app/src/main/java/org/simple/clinic/user/User.kt

```diff
@@ -0,0 +1,74 @@
+package org.simple.clinic.user
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.PrimaryKey
+import android.arch.persistence.room.Query
+import io.reactivex.Flowable
+import org.simple.clinic.util.RoomEnumTypeConverter
+import org.threeten.bp.Instant
+import java.util.UUID
+
+// We intentionally have to keep the table name same because
+// Room starts complaining if you try to rename a table which is referenced
+// by another table in a foreign key
+// (technically, SQLite supports renaming tables, but Room complains)
```

Comment:
Nitpick: Can I also ask you to use periods in comments?

Reply:
Nitpick: Justified text looks even better:

```
// We intentionally have to keep the table name same because Room 
// starts complaining if you try to rename a table which is referenced
// by another table in a foreign key. Technically, SQLite supports 
// renaming tables, but Room complains.
```